scheduler/src/cook/mesos.clj

```diff
@@ -32,54 +32,13 @@
             [datomic.api :as d :refer (q)]
             [mesomatic.scheduler]
             [mesomatic.types]
-            [metatransaction.core :as mt :refer (db)]
-            [metatransaction.utils :as dutils]
             [metrics.counters :as counters]
             [swiss.arrows :refer :all])
   (:import [org.apache.curator.framework.recipes.leader LeaderSelector LeaderSelectorListener]
            org.apache.curator.framework.state.ConnectionState))
 
 ;; ============================================================================
 ;; mesos scheduler etc.
-
-(defn submit-to-mesos
```

Comment:
AFAICT, this is dead code; the only use being to create the fn stored under :submitter below. However, I find no references of :submitter anywhere. This appears to be a helper function for batch submission by clients that write to the DB directly, and wired to metatransaction via dutil/transact-with-retries!!.